.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,19 @@
-<!-- describe the changes you have made here: what, why, ... 
-     Link issues by using the following pattern: [#333](https://github.com/JabRef/jabref/issues/333) or [koppor#49](https://github.com/koppor/jabref/issues/47).
-     The title of the PR must not reference an issue, because GitHub does not support autolinking there. -->
+<!-- 
+Describe the changes you have made here: what, why, ... 
+Link issues by using the following pattern: #333.
+If you fixed a koppor issue, link it with following pattern: [koppor#47](https://github.com/koppor/jabref/issues/47).
+The title of the PR must not reference an issue, because GitHub does not support autolinking there.
+-->
 
+## Checklist
 
-----
+I went through the following checklist and checked all items. All items with `[ ]` are still a TODO. All items checked with `[x]` are done.
 
-- [ ] Change in CHANGELOG.md described
-- [ ] Tests created for changes
-- [ ] Manually tested changed features in running JabRef
+I added new TODOs at the end of the list (if applicable).
+
+- [ ] Change in CHANGELOG.md described (if applicable)
+- [ ] Tests created for changes (if applicable)
+- [ ] Manually tested changed features in running JabRef (always required)
 - [ ] Screenshots added in PR description (for bigger UI changes)
 - [ ] Ensured that [the git commit message is a good one](https://github.com/joelparkerhenderson/git_commit_message)
```

Comment:
The git message doesn't really add anything. If I'm proposing a PR than the commits are already down. This can be added as a helpful comment at the end (for the next time).

Reply:
Yeah, we should move that to our contribution guideline